<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessInfoClass</class>
 <widget class="QMainWindow" name="ProcessInfoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ProcessInfo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ProcessInfo.qrc">
    <normaloff>:/ProcessInfo.ico</normaloff>:/ProcessInfo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::WidgetWidth</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProcesses">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionProcess_mode"/>
    <addaction name="actionModule_mode"/>
    <addaction name="separator"/>
    <addaction name="actionVMmap_tool"/>
    <addaction name="separator"/>
    <addaction name="actionSystem_processes"/>
   </widget>
   <addaction name="menuProcesses"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionProcess_mode">
   <property name="text">
    <string>Process mode</string>
   </property>
  </action>
  <action name="actionModule_mode">
   <property name="text">
    <string>Module mode</string>
   </property>
  </action>
  <action name="actionVMmap_tool">
   <property name="text">
    <string>VMmap tool</string>
   </property>
  </action>
  <action name="actionSystem_processes">
   <property name="text">
    <string>System processes</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ProcessInfo.qrc"/>
 </resources>
 <connections/>
</ui>
